### DISCUSSION TITLE
Border "right" over border "bottom"

### DISCUSSION THREAD

**Discussion by AregShahbazian** [caplin/FlexLayout#407] [Q&A]
The tab content of the right border is covered with that of the bottom border. 
Is it possible to make the right border be on top?
I couldn't find any attribute or props to make this happen.

![right-over-bottom](https://github.com/caplin/FlexLayout/assets/3205584/08c20950-bf12-44e8-b23d-43a414bde4a7)

**Comment by nealus** [comment]
try changing the order of the borders in the intial json

**Comment by AregShahbazian** [comment]
That did it.
Thanks!
